### PR TITLE
Bestiary monsters

### PR DESCRIPTION
**Pull request by ArturKnopik**
Continuation of https://github.com/otland/forgottenserver/pull/4356

Some notes for current content: 
- locations - created based on monster name + " locations" (ex: "Burning Book locations")
- Affected monsters - currently from monsters starting with the letters A to C (file name)
- current affected files - 95 monsters
- highest raceId value - 1824 for Cobra Vizier (12.20 update), can affect bestiary storage range

### Pull Request Prelude

- [X] I have followed [proper The Forgotten Server code styling][code].
- [X] I have read and understood the [contribution guidelines][cont] before making this PR.
- [X] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.

### CODE REVIEW

File: data/monster/monsters/acid_blob.xml [hunk 1]
```diff
@@ -18,6 +18,8 @@
 		<flag canwalkonenergy="0" />
 		<flag canwalkonfire="0" />
 	</flags>
+	<bestiary class="Slime" prowess="50" expertise="500" mastery="1000"
+		charmPoints="25" stars="3" occurrence="0" locations="Acid Blob locations"/>
```
Comment (EPuncker) [inline data/monster/monsters/acid_blob.xml:22]:
```suggestion
		charmPoints="25" stars="3" occurrence="0" locations="Acid Blob locations" />
```
same for all other

File: data/monster/monsters/coral_frog.xml [hunk 2]
```diff
@@ -19,6 +19,8 @@
 		<flag canwalkonfire="0" />
 		<flag canwalkonpoison="0" />
 	</flags>
+	<bestiary class="Aquatic" prowess="25" expertise="250" mastery="500"
```
Comment (torresmon) [inline data/monster/monsters/coral_frog.xml:22]:
```suggestion
	<bestiary class="Amphibic" prowess="25" expertise="250" mastery="500"
```

File: data/monster/monsters/crimson_frog.xml [hunk 3]
```diff
@@ -19,6 +19,8 @@
 		<flag canwalkonfire="0" />
 		<flag canwalkonpoison="0" />
 	</flags>
+	<bestiary class="Aquatic" prowess="25" expertise="250" mastery="500"
```
Comment (torresmon) [inline data/monster/monsters/crimson_frog.xml:22]:
```suggestion
	<bestiary class="Amphibic" prowess="25" expertise="250" mastery="500"
```

File: data/monster/monsters/crocodile.xml [hunk 4]
```diff
@@ -19,6 +19,8 @@
 		<flag canwalkonfire="0" />
 		<flag canwalkonpoison="0" />
 	</flags>
+	<bestiary class="Aquatic" prowess="25" expertise="250" mastery="500"
```
Comment (torresmon) [inline data/monster/monsters/crocodile.xml:22]:
```suggestion
	<bestiary class="Reptile" prowess="25" expertise="250" mastery="500"
```

File: data/monster/monsters/adept_of_the_cult.xml [hunk 5]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<monster name="Adept of the Cult" nameDescription="an adept of the cult" race="blood" experience="400" speed="215">
+<monster name="Adept of the Cult" nameDescription="an adept of the cult" race="blood" experience="400" speed="215" raceId="253">
```
Comment (ranisalt) [inline data/monster/monsters/adept_of_the_cult.xml:2]: Duplicate from above

File: data/monster/monsters/acid_blob.xml [hunk 6]
```diff
@@ -18,6 +18,8 @@
 		<flag canwalkonenergy="0" />
 		<flag canwalkonfire="0" />
 	</flags>
+	<bestiary class="Slime" prowess="50" expertise="500" mastery="1000"
+		charmPoints="25" stars="3" occurrence="0" locations="Acid Blob locations" />
```
Comment (ranisalt) [inline data/monster/monsters/acid_blob.xml:22]: Just leave locations empty, it should send "?" to the client

**Pull request closed without merging**